packages/core/core/package.json

```diff
@@ -18,7 +18,8 @@
     "vue": "^2.6.11",
     "lodash-es": "^4.17.15",
     "express": "^4.17.1",
-    "axios": "0.21.1"
+    "axios": "0.21.1",
+    "fs-extra": "^9.1.0"
```

Comment:
to be removed

Reply:
Why? You are using it in `core/core/src/server/memory/index.ts`